### DISCUSSION TITLE
UpdateFrame (shaders.input) unable to work on AMD but works on INTEL

### DISCUSSION THREAD

**Discussion by Mardornack** [bonsai-rx/bonsai#1299] [Q&A]
Hello! 
I am currently trying to make a workflow to study mouse movement in Parkinson disease. Unfortunatly the UpdateFrame (that allows me to track 2 PC mice simultaneously) does not work with my computer processor AMD. 
Is there a solution for this issue or the only answer is to switch to INTEL?
Best regards
Lisandro

**Comment by bruno-f-cruz** [comment]
Uhm that's bizarre. We have run the shaders package with AMD in the past. Can you share your workflow, computer specs and the error you are getting? Thanks

**Reply by Mardornack** [reply]
Yes sure. I have try the input in multiple computers with INTEL and AMD and the results were always the same. The error is the following: The UpdateFrame does not open up when you double click it before starting the workflow, therefore it can not be setted and activated. 
[TwoMice.zip](https://github.com/bonsai-rx/bonsai/files/11031909/TwoMice.zip)
or just see it here
![Image](https://user-images.githubusercontent.com/128514957/226695658-d797df5b-4b9c-4f1e-89c4-6ca2bfbbf059.svg)
Thanks a lot for your rapid response!

**Reply by bruno-f-cruz** [reply]
You need to create a context by adding a "CreateWindow" node to the top of the workflow.

**Reply by Mardornack** [reply]
That was it! Thanks a lot Bruno!
